api-samples/README.md

```diff
@@ -0,0 +1,6 @@
+Before running the code samples, ensure your development environment is
+properly configured. Consult the Google Surveys [Sample Code][1] page for
+setup instructions.
+
+[1]: https://developers.google.com/surveys/v2/samples
```

Comment:
This page just links to the getting started guide for more detailed directions. Perhaps we should just link there directly?